src/views/projectPageWebsite.hbs

```diff
@@ -12,14 +12,14 @@
   </h1>
   <ul class="information-ui">
     <li> {{res.projectResult.0.cohort_name}} </li>
-    <li><a target="_blank" href="{{res.projectResult.0.githublink}}">{{res.projectResult.0.githublink}}</a> </li>
-    <li><a target="_blank" href="{{res.projectResult.0.websitelink}}">{{res.projectResult.0.websitelink}}</a> </li>
+    <li><a target="_blank" href="{{res.projectResult.0.githublink}}">Github Link</a></li>
+    <li><a target="_blank" href="{{res.projectResult.0.websitelink}}">Website Link</a> </li>
     <li>
       students
       <ul class="sutdentsUl">
         {{#if res.projectStudentsResult.length}}
           {{#each res.projectStudentsResult}}
-           <li><a target="_blank" href="{{github_link}}"> {{studentName}}</a></li>
+           <li><a target="_blank" href="{{githubLink}}"> {{studentName}}</a></li>
           {{/each}}
         {{else}}
           <h3>There are No Student !</h3>
```

Comment:
Fix indentation.

Reply:
@NoureldeanSaed which line this indentation problem ??!! i can't find it !!